src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -61,3 +61,22 @@ export function guessExtensionFromHref(url) {
   }
   return detectedExtension;
 }
+
+/**
+ * Finds if a fragment contains a control group.
+ * NOTE: Control groupings will contain two letters, followed by a "-" and a positive integer
+ *
+ * @param frag A given fragment.
+ * @returns A string providing the control group.
```

Comment:
What does this mean... "A string providing the control group"

Reply:
Do I need to go more in depth on what a control group is or rephrase this returns comment? I'm just saying: "the function returns a string that is a control group."

Reply:
I'll add in the word "title" - or honestly "id" probably fits better. Is word choice of "providing" not clear? I've commonly written my return statements like this, including other places in this file.

Reply:
I'd update the other ones too unless Kyle disagrees. I think it just sounds a bit awkward. I think the string just is `the control group id` right? The string doesn't "provide" us anything.

I realize this is a bit pedantic, but it just sounds weird 😄

Reply:
"providing" is very confusing -- "Control group ID" makes much more sense. And @tuckerzp is correct that we should put the type in braces. So:

```
@param {string} the control group ID
```

is probably most correct. Please correct this occurrence and any others.

Reply:
I want to make sure I get this right. `@param` we add braces too around the type, but `@returns` we don't, right? I'm noticing most returns we don't explicitly say the type on

Reply:
I personally prefer stating the type for the return when we can. I would be hesitant to base decisions on some of our past coding practices.

Reply:
It is something I want to start doing and to improve as we go forward. Just because we haven't _always_ done the best/correct thing does not mean we should avoid iterative improvement. https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

Reply:
Created an issue to revise our previous JSDoc: https://github.com/EasyDynamics/oscal-react-library/issues/724